crypto.c

```diff
@@ -33,13 +33,19 @@
  * SUCH DAMAGE.
  */
 
+#include <openssl/opensslv.h>
```

Comment:
There were already uses of `OPENSSL_VERSION_NUMBER` prior to your change - curious why openssl/opensslv.h was not necessary before. Did it come in via transitive dependencies before?